drakcore/drakcore/app.py

```diff
@@ -65,6 +67,35 @@ def logs(task_uid, log_type):
         return send_file(f.name, mimetype='text/plain')
 
 
+def processed_generic(func, obj_name: str, mime: str = "application/json"):
+    def wrapper(task_uid: str):
+        gen_name = f"{task_uid}/processed/{obj_name}"
+        try:
+            # Fast path: object exists
+            with NamedTemporaryFile() as tmp_file:
+                minio.fget_object("drakrun", gen_name, tmp_file.name)
+                return send_file(tmp_file.name, mime)
+        except NoSuchKey:
+            pass
+        # Slow path: generate and upload to minio
+        data = func(minio, task_uid)
+        file = BytesIO(data)
+        minio.put_object("drakrun", gen_name, file, length=len(data))
+        return Response(data, mimetype=mime)
+
+    return wrapper
+
+
+processed_generators = {
+    "process_tree.json": processed_generic(generate_process_tree, "process_tree.json"),
+}
+
+
+@app.route("/processed/<task_uid>/<which>")
+def processed(task_uid, which):
+    return processed_generators[which](task_uid)
```

Comment:
such post processing artifacts are usually generated by drakrun itself

somebody may just look at the output directory at minio storage and not use drakcore HTTP API at all

this hovewer needs further design so for now I think is OK